ansible_rulebook/builtin.py

```diff
@@ -786,8 +788,20 @@ async def shutdown(
             rule=source_rule_name,
             run_at=str(datetime.utcnow()),
             matching_events=_get_events(variables),
+            delay=delay,
+            message=message,
         )
     )
+
+    logger.info(
```

Comment:
I see a problem here: the message is printed in the stdout only if --verbose or --debug flags are enabled. This would not resolve one of the problems described in the ticket related with the user experience. 

This message, as well as other actions, should be printed to the stdout by the default.

Reply:
@Alex-Izquierdo I have changed it to a print statement now.
There is more work that needs to be done around Shutdown based on the changes discussed in PR #296 
We will be doing that in a different PR.